js/collections/VerifiedMods.js

```diff
@@ -18,7 +19,8 @@ export default class extends Collection {
   }
 
   url() {
-    return app.localSettings.get('verifiedModsProvider');
+    const usingTor = app.serverConfig.tor && getCurrentConnection().server.get('useTor');
```

Comment:
I think you could eliminate the `app.serverConfig.tor` part there since it doesn't really matter if the server is using tor or not because this call wouldn't go through the server. `getCurrentConnection().server.get('useTor')` indicates that the user chose to use tor in their server config and that IMHO should be enough for us to route the verified mods call through a tor endpoint.